sdks/js/src/types.ts

```diff
@@ -124,6 +124,21 @@ const SupportedContentFragmentTypeSchema = FlexibleEnumSchema([
   ...(Object.keys(supportedLegacy) as [keyof typeof supportedLegacy]),
 ]);
 
+const uniq = <T>(arr: T[]): T[] => Array.from(new Set(arr));
```

Comment:
`nit` you can use lodash `uniqBy`.

Reply:
I don't want to add a dependancy just for that.